dspace-server-webapp/src/test/java/org/dspace/app/rest/CollectionGroupRestControllerIT.java

```diff
@@ -0,0 +1,2022 @@
+                        .andExpect(status().isNoContent());
+    }
+
+
+    // This is currently not supported in DSpace API
+    @Ignore
+    @Test
+    public void deleteCollectionSubmittersGroupTestParentCommunityAdmin() throws Exception {
+        Group submittersGroup = collectionService.createSubmitters(context, collection);
+        authorizeService.addPolicy(context, parentCommunity, Constants.ADMIN, eperson);
+
+        String token = getAuthToken(eperson.getEmail(), password);
+        getClient(token).perform(delete("/api/core/collections/" + collection.getID() + "/submittersGroup"))
+                        .andExpect(status().isNoContent());
+
+        getClient(token).perform(get("/api/core/collections/" + collection.getID() + "/submittersGroup"))
+                        .andExpect(status().isNoContent());
+    }
+
+    // This is currently not supported in DSpace API
+    @Ignore
+    @Test
+    public void deleteCollectionSubmittersGroupTestCollectionAdmin() throws Exception {
+        Group submittersGroup = collectionService.createSubmitters(context, collection);
+        authorizeService.addPolicy(context, collection, Constants.ADMIN, eperson);
+
+        String token = getAuthToken(eperson.getEmail(), password);
+        getClient(token).perform(delete("/api/core/collections/" + collection.getID() + "/submittersGroup"))
+                        .andExpect(status().isNoContent());
+
+        getClient(token).perform(get("/api/core/collections/" + collection.getID() + "/submittersGroup"))
+                        .andExpect(status().isNoContent());
+    }
+
+    @Test
+    public void deleteCollectionSubmittersGroupUnAuthorizedTest() throws Exception {
+        Group submittersGroup = collectionService.createSubmitters(context, collection);
+
+
+        getClient().perform(delete("/api/core/collections/" + collection.getID() + "/submittersGroup"))
+                   .andExpect(status().isUnauthorized());
+    }
+
+    @Test
+    public void deleteCollectionSubmittersGroupForbiddenTest() throws Exception {
+        Group submittersGroup = collectionService.createSubmitters(context, collection);
+
+        String token = getAuthToken(eperson.getEmail(), password);
+
+        getClient(token).perform(delete("/api/core/collections/" + collection.getID() + "/submittersGroup"))
+                        .andExpect(status().isForbidden());
+    }
+
+
+    @Test
+    public void deleteCollectionSubmittersGroupNotFoundTest() throws Exception {
+        Group submittersGroup = collectionService.createSubmitters(context, collection);
+
+        String token = getAuthToken(eperson.getEmail(), password);
+
+        getClient(token).perform(delete("/api/core/collections/" + UUID.randomUUID() + "/submittersGroup"))
+                        .andExpect(status().isNotFound());
+    }
+
+    private Group createGroup(String itemGroupString, int defaultItemRead) throws SQLException, AuthorizeException {
```

Comment:
This method appears to be misnamed. It's only creating *_DEFAULT_READ groups, and not just any group.  It probably should be named `createDefaultReadGroup` or similar so that its purpose is clear.